lib/Test/Selenium/Remote/Driver.pm

```diff
@@ -126,15 +132,20 @@ See L<Selenium::Remote::Driver> for the meanings of these options.
 sub BUILDARGS {
     my ( undef, %p ) = @_;
 
+    OPT:
     for my $opt (
         qw/remote_server_addr port browser_name version platform
         javascript auto_close extra_capabilities/
       )
     {
-        $p{$opt} //= $ENV{ 'TWD_' . uc($opt) };
+        my $env_var_name = 'TWD_' . uc($opt);
+
+        next OPT unless exists $ENV{$env_var_name};
+
+        $p{$opt} //= $ENV{$env_var_name};
     }
-    $p{browser_name}       //= $ENV{TWD_BROWSER};                      # ykwim
-    $p{remote_server_addr} //= $ENV{TWD_HOST};                         # ykwim
+    $p{browser_name}       //= $ENV{TWD_BROWSER} if exists $ENV{TWD_BROWSER};  # ykwim
```

Comment:
It may be worth mentioning these two aliases have lowest precedence, I'm sure someone will be careless enough to use both the fully qualified name and the alias simultaneously.